test/operator-hub/subscription.yaml

```diff
@@ -2,10 +2,11 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: my-service-binding
+  name: service-binding-operator
   namespace: openshift-operators
 spec:
-  channel: dev
-  name: service-binding
-  source: my-catalog
-  sourceNamespace: openshift-operator-lifecycle-manager
+  channel: alpha
+  installPlanApproval: Automatic
+  name: service-binding-operator
+  source: example-operators
```

Comment:
In `./test/operator-hub/operator_source.yaml`  the name is given as `my-catalog`. Can you please confirm if this is correct? Also, there is a `REPLACE_IMAGE` in that file. When is that value getting replaced?

Reply:
Please ignore the above comment. That file doesn't exist. But it is referred from Makefile.